contracts/staking/stakeManager/IStakeManager.sol

```diff
@@ -1,23 +1,8 @@
 pragma solidity ^0.5.2;
 
-contract IStakeManager {
-    // Todo: fix WITHDRAWAL_DELAY with interface
-    uint256 public WITHDRAWAL_DELAY = (2**13); // unit: epoch
-    uint256 public currentEpoch = 1;
+import {StakeManagerStorage} from "./StakeManagerStorage.sol";
 
-    enum Status {Inactive, Active, Locked, Unstaked}
-    struct Validator {
-        uint256 amount;
-        uint256 reward;
-        uint256 activationEpoch;
-        uint256 deactivationEpoch;
-        uint256 jailTime;
-        address signer;
-        address contractAddress;
-        Status status;
-    }
-
-    mapping(uint256 => Validator) public validators;
+contract IStakeManager is StakeManagerStorage {
```

Comment:
Why does the interface need to inherit from storage?

Reply:
Using some variables as public functions and this interface as well. so all the contracts using `IStakeManager` can read public variables and use interface functions.